etl/transformer/transformer.go

```diff
@@ -186,7 +186,7 @@ func getPulseData(pn *exporter.FullPulse) types.Pulse {
 	return types.Pulse{
 		PulseNo:         int64(pulse.AsUint32()),
 		EpochPulseNo:    int64(pulse.AsEpoch()),
-		PulseTimestamp:  pn.GetPulseTimestamp(),
+		PulseTimestamp:  pn.GetPulseTimestamp() / 1000000000,
```

Comment:
maybe move to separate function if possible?

Reply:
I don't think so because here is the one place where it needs to do.